### PR TITLE
New link param

### PR DESCRIPTION
**Pull request by Balachandark**

### CODE REVIEW

File: inc/extras.php [hunk 1]
```diff
@@ -437,28 +437,30 @@ function astra_get_custom_html( $option_name = '' ) {
 	 *
 	 * @since 1.0.0
 	 * @param  string $button_text Button Text.
-	 * @param  string $button_link Button Link.
+	 * @param  array  $button_options Button Link.
```
Comment (premanshup) [inline inc/extras.php]: @Balachandark This should be `string` instead of `array`

Comment (Balachandark) [reply inc/extras.php]: @premanshup $button_options is an array with details of URL, new_tab and rel

File: inc/customizer/custom-controls/class-astra-customizer-control-base.php [hunk 2]
```diff
@@ -58,6 +58,9 @@ public function enqueue_scripts() {
 			wp_enqueue_style( 'custom-control-style' . $file_rtl, $css_uri . 'custom-controls' . $file_prefix . $file_rtl . '.css', null, ASTRA_THEME_VERSION );
 			wp_enqueue_script( 'custom-control-script', $js_uri . 'custom-controls' . $file_prefix . '.js', array( 'jquery', 'customize-base', 'astra-color-alpha', 'jquery-ui-tabs', 'jquery-ui-sortable' ), ASTRA_THEME_VERSION, true );
 
+			// delete this before opening PR.
+			wp_enqueue_script( 'custom-control-script-testing', ASTRA_THEME_URI . 'inc/customizer/custom-controls/link/link.js', array( 'jquery', 'customize-base', 'astra-color-alpha', 'jquery-ui-tabs', 'jquery-ui-sortable' ), ASTRA_THEME_VERSION, true );
```
[the remark this answers is not in the document]
Comment (Balachandark) [reply inc/customizer/custom-controls/class-astra-customizer-control-base.php]: Testing code removed

File: inc/extras.php (hunk 1 above)
Comment (premanshup) [reply inc/extras.php]: @Balachandark As per the discussion, I have made the changes.

**Pull request merged**